### PR TITLE
{bio}[foss/2022a] NanoPlot v1.42.0, nanoget v1.19.1, pauve v0.2.3, and use nanoget 1.19.1 for Nanostat 1.6.0

### PR DESCRIPTION
**Pull request by moravveji**
This PR adds three easyconfigs for building `pauvre/0.2.3-foss-2022a`, `nanoget/1.19.1-foss-2022a` and eventually `NanoPlot/1.42.0-foss-2022a`.

### CODE REVIEW

**Comment by moravveji** [comment]
Apparently, the test builds fail with the following error:
```
FAIL: test_dep_versions_per_toolchain_generation (test.easyconfigs.easyconfigs.EasyConfigTest.test_dep_versions_per_toolchain_generation)
Check whether there's only one dependency version per toolchain generation actively used.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/easybuild-easyconfigs/easybuild-easyconfigs/test/easyconfigs/easyconfigs.py", line 957, in test_dep_versions_per_toolchain_generation
    self.fail('Should not have multiple variants of dependencies.\n' + multi_dep_vars_msg)
AssertionError: Should not have multiple variants of dependencies.
Found 2 variants of 'nanoget' dependency in easyconfigs using '2022a' toolchain generation
* version: 1.18.1; versionsuffix:  as dep for {'NanoStat-1.6.0-foss-2022a.eb'}
  * version: 1.19.1; versionsuffix:  as dep for {'NanoPlot-1.42.0-foss-2022a.eb'}
 ```
In my case, `nanoget` version `1.19.1` is needed as the minimum requirement for `NanoPlot`. That's why I eneded up having two versions of `nanoget` under one toolchain.
Is there a way to resolve this?

**Comment by migueldiascosta** [comment]
@moravveji that test is meant to avoid a proliferation of dependency variants in the same toolchain generation, but there are a few of options:

- bump this PR to a more recent toolchain version 
- add an exception for nanoget in `check_dep_vars` in `easybuild-easyconfigs/test/easyconfigs/easyconfigs.py`
- since that older version of `nanoget` is only used in `NanoStat-1.6.0-foss-2022a.eb` and that `NanoStat` is not a dependency for anything else, I guess one could also bump the `nanoget` version there, if supported...

**Comment by moravveji** [comment]
> * since that older version of `nanoget` is only used in `NanoStat-1.6.0-foss-2022a.eb` and that `NanoStat` is not a dependency for anything else, I guess one could also bump the `nanoget` version there, if supported...

thanks @migueldiascosta for the tip. I have just followed your suggestion in https://github.com/easybuilders/easybuild-easyconfigs/pull/20338/commits/ced6b7519f515eb38b06d45747bea5dea7635eac and now the tests succeed, apparently.

File: easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.42.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('Biopython', '1.79'),
+    ('Pysam', '0.19.1'),
+    ('nanomath', '1.3.0'),
+    ('nanoget', '1.19.1'),
+    ('Seaborn', '0.12.1'),
+    ('plotly.py', '5.12.0'),
+    ('pauvre', '0.2.3'),
```
Comment (bedroge) [inline easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.42.0-foss-2022a.eb:26]: I'm wondering if this still needs Seaborn and pauvre: the setup.py of this version (https://github.com/wdecoster/NanoPlot/blob/v1.42.0/setup.py#L34) doesn't list them anymore, while the one for 1.3.0 did (https://github.com/wdecoster/NanoPlot/blob/1.33.0/setup.py#L43). There's a specific commit (https://github.com/wdecoster/NanoPlot/commit/f977a26ff9f3c6e76d80e2c10f36c95a4d603457) where they're dropping them, and one where the `pauvre_plot` function gets commented (https://github.com/wdecoster/NanoPlot/commit/17bde314ca69776246d9ecab8639b9dee9832730).

Comment (moravveji) [reply easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.42.0-foss-2022a.eb]: Absolutely right. https://github.com/easybuilders/easybuild-easyconfigs/pull/20338/commits/e81fefba03d96b544b1b24eefb543dba4c654ae4 shall address that.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/124a733f61a45cafdf62565d84746f70 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/22ef895dc2643ba5b87ce0ad7a1375f2 for a full test report.

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Going in, thanks @moravveji!

**Pull request merged**